### PR TITLE
 EN-2313 Implemented rating on engine side

### PR DESCRIPTION
**Pull request by zonia3000**

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/digitalexchange/rating/DEComponentRatingResult.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.entando.entando.web.common.model.RestError;
+
+public class DEComponentRatingResult {
+
+    private boolean ratingUnsupported;
```
[the remark this answers is not in the document]
Comment (zonia3000) [reply plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/aps/system/services/digitalexchange/rating/DEComponentRatingResult.java]: Done

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

**Pull request merged**